sleap_nn/training/lightning_modules.py

```
@@ -584,3 +584,1300 @@ def validation_step(self, batch, batch_idx):
+            self.model.backbone.enc.load_state_dict(ckpt, strict=False)
+
+        # TODO: Handling different input channels
+        # Initializing backbone (encoder + decoder) with trained ckpts
+        if self.pretrained_backbone_weights is not None:
+            logger.info(
+                f"Loading backbone weights from `{self.pretrained_backbone_weights}` ..."
+            )
+            ckpt = torch.load(self.pretrained_backbone_weights)
+            ckpt["state_dict"] = {
+                k: ckpt["state_dict"][k]
+                for k in ckpt["state_dict"].keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)